src/compartment-definition/PatientReferences.json

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
I know that the PR description says that the sources of this is the compartmentdefinition parsing done with scripts in deqm-test-server. Do you think there's somewhere we can explicitly site that in either comments (stupid json not allowing comments) or fqm-execution documentation? If compartment definition stuff changes in the future, I think it's valuable to have as much of a thread as possible for future developers to be able to make the updates needed.

Reply:
we could make this a `.ts` file and do like `export const PatientReferences = {...}`?? Then we could include any comments/citations at the top of the file or something